src/traces/parcoords/base_plot.js

```diff
@@ -28,8 +27,6 @@ exports.clean = function(newFullData, newFullLayout, oldFullData, oldFullLayout)
     var hasParcoords = (newFullLayout._has && newFullLayout._has('parcoords'));
 
     if(hadParcoords && !hasParcoords) {
-        oldFullLayout._paperdiv.selectAll('.parcoords-line-layers').remove();
-        oldFullLayout._paperdiv.selectAll('.parcoords-line-layers').remove();
```

Comment:
Where is this done now?

Reply:
Since we keep canvases, I guessed we have to store them instead of recreating every time.
Actually removing and creating anew would solve the problem...